### ISSUE TITLE
Sway is broken on unstable

### ISSUE THREAD

**Issue by lovesegfault** [colemickens/nixpkgs-wayland#174]
```
unpacking channels...
building Nix...
building the system configuration...
error: while evaluating the attribute 'activationScript' of the derivation 'nixos-system-foucault-20.03pre203787.28cb34d538d' at /nix/var/nix/profiles/per-user/root/channels/nixos/nixos/modules/system/activation/top-level.nix:102:5:
while evaluating the attribute 'system.activationScripts.script' at /nix/var/nix/profiles/per-user/root/channels/nixos/nixos/modules/system/activation/activation-script.nix:68:9:
while evaluating 'textClosureMap' at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/strings-with-deps.nix:70:35, called from /nix/var/nix/profiles/per-user/root/channels/nixos/nixos/modules/system/activation/activation-script.nix:89:18:
while evaluating 'id' at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/trivial.nix:14:5, called from undefined position:
while evaluating the attribute 'text' at /nix/var/nix/profiles/per-user/root/channels/nixos/nixos/modules/system/activation/activation-script.nix:9:5:
while evaluating the attribute 'text' at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/strings-with-deps.nix:77:38:
while evaluating the attribute 'sources' of the derivation 'etc' at /nix/var/nix/profiles/per-user/root/channels/nixos/nixos/modules/system/etc/etc.nix:12:5:
while evaluating anonymous function at /nix/var/nix/profiles/per-user/root/channels/nixos/nixos/modules/system/etc/etc.nix:20:20, called from undefined position:
while evaluating the attribute 'source' at undefined position:
while evaluating anonymous function at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:75:45, called from undefined position:
while evaluating the attribute 'value' at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:338:9:
while evaluating the option `environment.etc.sway/config.source':
while evaluating the attribute 'isDefined' at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:375:5:
while evaluating the attribute 'values' at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:364:9:
while evaluating anonymous function at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:360:19, called from /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:360:14:
while evaluating the attribute 'value._type' at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:456:73:
while evaluating the attribute 'value.content' at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:517:14:
while evaluating the attribute 'sway' at /nix/store/rq2f1kjlfrk9l7sfm2kln5j5qik4jnig-source/default.nix:15:3:
while evaluating 'callPackageWith' at /nix/store/nyaf3smjvipcdqx0righ98m1fgf7h21z-nixos-20.03pre203787.28cb34d538d/nixos/lib/customisation.nix:117:35, called from /nix/store/rq2f1kjlfrk9l7sfm2kln5j5qik4jnig-source/default.nix:15:22:
while evaluating 'makeOverridable' at /nix/store/nyaf3smjvipcdqx0righ98m1fgf7h21z-nixos-20.03pre203787.28cb34d538d/nixos/lib/customisation.nix:67:24, called from /nix/store/nyaf3smjvipcdqx0righ98m1fgf7h21z-nixos-20.03pre203787.28cb34d538d/nixos/lib/customisation.nix:121:8:
anonymous function at /nix/store/rq2f1kjlfrk9l7sfm2kln5j5qik4jnig-source/pkgs/sway/default.nix:1:1 called without required argument 'libGLU_combined', at /nix/store/nyaf3smjvipcdqx0righ98m1fgf7h21z-nixos-20.03pre203787.28cb34d538d/nixos/lib/customisation.nix:69:16
```

**Comment by lovesegfault** [comment]
The issue seems to come from libGLU_combined, which no longer exists
```
nixpkgs on  master took 6s
❯ rg -i libGLU_combined
```

**Comment by colemickens** [comment]
Sorry about this. I've got this issue fixed, but `wldash` was broken by the `nixpkgs` update without `wldash` itself updating. I'm trying to track it down. I might back out `wldash` temporarily to unblock this overlay. (Also, Sway just added a nice improvement for HiDPI users that might be nice to get out).

**Comment by colemickens** [comment]
see: https://github.com/NixOS/nixpkgs/issues/74812

**Comment by lovesegfault** [comment]
It'd be nice to back-out wldash for now, IMO

**Comment by colemickens** [comment]
Oof, I should've just pushed this, there's no reason not to. It unbreaks everything else but just leaves wldash at the mercy of whatever changed in nixpkgs until I can track it down. I'll get this pushed and close this.

**Comment by colemickens** [comment]
Fixed in  947862e93cf9d69bd0bde2153acf9b96673be7f8.
